php/WC_Filter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+defined( 'ABSPATH' ) or exit;
+add_action( 'plugins_loaded', 'wc_filter_orders_by_bitcoin_address' );
+
+class WC_Filter_Orders_By_Address {
+	protected static $instance;
+	public function __construct() {
+		if ( is_admin() ) {
+			// add bulk order filter for exported / non-exported orders
+			add_action( 'restrict_manage_posts', array( $this, 'filter_orders') , 20 );
+			add_filter( 'request',               array( $this, 'filter_orders_by_address_or_txid' ) );	
+
+		}
+	}
+	public function filter_orders() {
+		global $typenow;
+		if ( 'shop_order' === $typenow ) {
+			?>
+			<input size='26' type='name' placeholder='Filter by crypto address/txid' name='filter_by'>
```

Comment:
When the filter is applied, the page refreshes and clears the address/txid from the input. We can use `value="<?php if(isset( $_GET['filter_by'] )) echo($_GET['filter_by']); ?>"` to set the value of the input to display the filtered address/txid.

Reply:
Great idea! I've added it.